app/components/cms/community_activity_grid_component/community_activity_grid_component.html.erb

```diff
@@ -0,0 +1,62 @@
+<%= render GovGridRowComponent.new(additional_classes: "community-activity-grid-wrapper") do |row| %>
+  <%= row.with_column(:full) do %>
+    <div class="community-activity-grid">
+      <% if @title %>
```

Comment:
Title is a required parameter in the component, do we need the if statement?

Reply:
Both this and the following one are required in the component, but are not required in strapi, so they could come through as nil. I will update the component definition to reflect this better